<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QBat - Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="criticalGroup">
     <property name="title">
      <string>Event for critical battery capacity</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="criticalCapacitylabel">
        <property name="text">
         <string>Critical capacity (in %)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="criticalWarningRadio">
        <property name="text">
         <string>Show warning message</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QRadioButton" name="criticalCommandRadio">
        <property name="text">
         <string>Execute command</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="criticalCapacitySpin">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QGroupBox" name="confirmCommandBox">
        <property name="title">
         <string>Confirm command</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="timeoutCheck">
           <property name="text">
            <string>Use timeout to confirm</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="timeoutSpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="timeoutLabel">
           <property name="text">
            <string>Seconds</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="spacer1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="criticalCommandEdit"/>
      </item>
      <item row="0" column="2">
       <spacer name="spacer0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="batteryIconGroup">
     <property name="title">
      <string>Battery icon(s)</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="5" rowspan="3">
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="mainColorLabel">
        <property name="text">
         <string>Main color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="mainFullColorButton">
        <property name="toolTip">
         <string>full</string>
        </property>
        <property name="text">
         <string>full</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="mainChargedColorButton">
        <property name="toolTip">
         <string>charged</string>
        </property>
        <property name="text">
         <string>charged</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="mainEmptyColorButton">
        <property name="toolTip">
         <string>empty</string>
        </property>
        <property name="text">
         <string>empty</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="textColorLabel">
        <property name="text">
         <string>Text color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="textFullColorButton">
        <property name="toolTip">
         <string>full</string>
        </property>
        <property name="text">
         <string>full</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="poleLabel">
        <property name="text">
         <string>&quot;+ pole&quot; color</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="poleFullColorButton">
        <property name="toolTip">
         <string>full</string>
        </property>
        <property name="text">
         <string>full</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="poleChargingColorButton">
        <property name="toolTip">
         <string>charging</string>
        </property>
        <property name="text">
         <string>charging</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QCheckBox" name="mergeBatteriesCheck">
        <property name="text">
         <string>Merge Icons</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="textChargingColorButton">
        <property name="toolTip">
         <string>charging</string>
        </property>
        <property name="text">
         <string>charging</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="textDischargingColorButton">
        <property name="toolTip">
         <string>discharging</string>
        </property>
        <property name="text">
         <string>discharging</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QToolButton" name="poleDischargingColorButton">
        <property name="toolTip">
         <string>discharging</string>
        </property>
        <property name="text">
         <string>discharging</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showBalloonCheck">
     <property name="text">
      <string>Show messages as balloon tips</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="pollingRateLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Polling rate (in milliseconds)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pollingRateSpin">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>20000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>437</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>criticalGroup</tabstop>
  <tabstop>criticalWarningRadio</tabstop>
  <tabstop>criticalCommandRadio</tabstop>
  <tabstop>mainFullColorButton</tabstop>
  <tabstop>mainChargedColorButton</tabstop>
  <tabstop>mainEmptyColorButton</tabstop>
  <tabstop>textFullColorButton</tabstop>
  <tabstop>textChargingColorButton</tabstop>
  <tabstop>poleFullColorButton</tabstop>
  <tabstop>poleChargingColorButton</tabstop>
  <tabstop>pollingRateSpin</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>criticalCommandRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>criticalCommandEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeoutCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeoutSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeoutCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>timeoutLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>criticalCommandRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>confirmCommandBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
